Reachability.m

```diff
@@ -152,6 +156,8 @@ -(instancetype)initWithReachabilityRef:(SCNetworkReachabilityRef)ref
         self.reachableOnWWAN = YES;
         self.reachabilityRef = ref;
 
+        CFRetain(self.reachabilityRef);
```

Comment:
My mistake. I missed the last ```CFRetain``` call on line 159 because of the way the inline comments split up the diff.